infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationUtils.kt

```diff
@@ -191,13 +191,29 @@ fun pointsAreSame(point1: IPoint?, point2: IPoint?) =
     point1 == point2 || point1 != null && point2 != null && point1.isSame(point2, DISTANCE_CHANGE_THRESHOLD)
 
 fun getLengthChangedRemarkOrNull(translation: Translation, length1: Double?, length2: Double?) =
-    if (length1 != null && length2 != null) lengthDifference(length1, length2).let { lengthDifference ->
-        if (lengthDifference > DISTANCE_CHANGE_THRESHOLD) publicationChangeRemark(
-            translation, "changed-x-meters", formatDistance(lengthDifference)
-        )
-        else null
+    if (length1 == null || length2 == null) {
+        null
+    } else {
```

Comment:
Voikohan meillä oikeasti olla tilannetta jossa jonkin julkaistun jutun pituus olisi koskaan null 🤔 Hyvähän tähän siis varautua on (varsinkin jos datatyypit sen mahdollistavat), rupesin vaan miettimään

Reply:
Muistelisin että deletoituna voi julkaista vaikkei geometriaa olisikaan, mutten ole ihan varma.

Reply:
Totta. Tästähän oli puhettakin hiljattain jonkun toisen jutun yhteydessä.

Reply:
👍 Nämä on vähän näitä nullien käyttöprobleemoita, ettei kontekstista ole selvää mitä nulli tarkoittaa ja mistä se on sinne päätynyt (ja ketju myös jatkuu, ei ole kovinkaan selvää minkä vuoksi nulli on päätynyt tämän funkkarin kutsujalle, kun tämäkin funkkari palauttaa nullin kolmesta eri paikasta).